data/releasenotes.json

```
@@ -1,5 +1,30 @@
 [
-		{
+	{
+		"version": "v2025-11-06",
+		"date": "2025-11-06",
+		"breaksdb": false,
+		"notes": [
+			"<b>New Feature:</b> Add pg_drop_caches() to clear buffer caches without restarting the database system",
+			"<b>New Feature:</b> Add support for RENAME COLUMN as part of ALTER TABLE",
+			"<b>New Feature:</b> Add support for sum(bool)",
+			"<b>New Feature:</b> Add unsigned types to SQL: uint1, uint2, uint4, uint8",
+			"<b>Performance:</b> Add improved compression for floating-point numbers (Adaptive Lossless Floating-Point Compression).",
+			"<b>Performance:</b> Faster comparison of EXTRACT (YEAR ...) with constants",
+			"<b>Performance:</b> Improved auto-compression of small inserts",
+			"<b>PostgreSQL Compatibility:</b> Add full support for PostgreSQL advisory locks, including the ability to wait until a resource is freed and detect deadlocks.",
```

I would categorize this as new feature, way to cool to hide under compatibility
```suggestion
			"<b>New Feature:</b> Add full support for PostgreSQL advisory locks, including the ability to wait until a resource is freed and detect deadlocks.",
```